### ISSUE TITLE
Can't process Microsoft.PowerShell.Commands.PSSessionConfigurationCommands#PSSessionConfiguration 

### ISSUE THREAD

**Issue by jdhitsolutions** [jdhitsolutions/PSTypeExtensionTools#10]
This command throws an exception and it shouldn't:
```
 Get-PSTypeExtension Microsoft.PowerShell.Commands.PSSessionConfigurationCommands#PSSessionConfiguration 
```

**Comment by jdhitsolutions** [comment]
Get-PSTypeExtension shouldn't throw errors:

```
get-typedata | where {$_.members.count -gt 0} | get-pstypeextension
```

**Comment by jdhitsolutions** [comment]
fixed in v0.5.0
